### PR TITLE
V0.0 skirt set (SSR, PowerSwitch, Nameplate and color change printing)

### PR DESCRIPTION
**Pull request by Zen3Dlab**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Screenshots (if appropriate)

img1

img2

### CODE REVIEW

File: printer_mods/Zen3D/V0_Skirt/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Voron0 dual color skirt set (SSR, PowerSwitch, Nameplate)
+
+To match the skirt with the [Midpanel with Hex Pattern](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/Zen3D/V0_MidPanel_HexPattern)  by sterminatore's idea
```
Comment (KiloQubit) [inline printer_mods/Zen3D/V0_Skirt/README.md:3]: Change this to a relative link

    [Midpanel with Hex Pattern](../V0_MidPanel_HexPattern)

File: printer_mods/Zen3D/V0_Skirt/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+![img3](./IMG/img1.jpg)
```
Comment (KiloQubit) [inline printer_mods/Zen3D/V0_Skirt/README.md:5]: you're using img1, but text says img3.

File: printer_mods/Zen3D/V0_Skirt/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+![img3](./IMG/img2.jpg)
```
Comment (KiloQubit) [inline printer_mods/Zen3D/V0_Skirt/README.md:7]: you're using img2, but text says img3.

File: printer_mods/Zen3D/V0_Skirt/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+## Skirt.stl, Skirt_mirror.stl
+
+Left and right skirt part with 40 degree chamfers for color change printing. There is an extra hole for tightening the middle M3 screw.
+
+![img5](./IMG/img5.jpg)
+
+## Skirt_EmptyNameplate.stl
+
+Empty nameplate for serial number. For example, with Tinkercad you can easily put the number on it.
```
Comment (KiloQubit) [inline printer_mods/Zen3D/V0_Skirt/README.md:22]: Can you add a tutorial (maybe at the end of this README or as a separate file and link to it) on how to add the number using Tinkercad, assuming the user has never used Tinkercad before.

File: printer_mods/Zen3D/V0_Skirt/README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+![img8](./IMG/img8.jpg)
+
+
+
+
+### Overview
+
+
+![img3](./IMG/img3.jpg)
```
Comment (KiloQubit) [inline printer_mods/Zen3D/V0_Skirt/README.md:46]: img0.jpg is not included in the README - either add it to the README, or remove the file

File: printer_mods/Zen3D/V0_Skirt/README.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+## Skirt_SSR.stl
+
+SSR holder with led window.
+
+![img7](./IMG/img7.jpg)
+
+## Skirt_PowerSwitch.stl
+
+12mm x 19mm windows for power switch.
```
Comment (KiloQubit) [inline printer_mods/Zen3D/V0_Skirt/README.md:36]: Add a link to an example power switch that fits into this skirt.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!  I just have some changes for the README below.

File: printer_mods/Zen3D/V0_Skirt/README.md (hunk 4 above)
Comment (Zen3Dlab) [reply printer_mods/Zen3D/V0_Skirt/README.md:22]: I made a clumsy video about using tinkercad. :-)

**Comment by Zen3Dlab** [comment]
@KiloQubit  I clicked on something and can no longer see your last comment. In principle, I fixed everything.

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**